### ISSUE TITLE
Please add safeupdate extension

### ISSUE THREAD

**Issue by t1u1** [supabase/supabase#6728]
# Feature request

`PostgREST` recommends installing the `safeupdate` extension:
https://postgrest.org/en/stable/admin.html#block-fulltable

This prevents accidental deletions when a query parameter is omitted by mistake.

Please add the option to install this extension.

**Comment by GaryAustin1** [comment]
pg-safeupdate | 1.4 | Protect your data from accidental updates or deletes.

Is installed by default already.

**Comment by t1u1** [comment]
Oh. But I don't see it listed in my project at: https://app.supabase.io/project/xyz/database/extensions

Is that expected?

**Comment by GaryAustin1** [comment]
Not sure, but delete with no where clause
![image](https://user-images.githubusercontent.com/54564956/167309561-d15330f1-28be-4f52-97d7-bff47d8810e3.png)

**Comment by liaujianjie** [comment]
I'm still able to delete without `WHERE` clause from the dashboard. Is that expected?

**Comment by steve-chavez** [comment]
> Oh. But I don't see it listed in my project at: https://app.supabase.io/project/xyz/database/extensions
> Is that expected?
> I'm still able to delete without WHERE clause from the dashboard. Is that expected?

Yes and yes. You enable `safeupdate` with `LOAD 'safeupdate'`(you can do it in the dashboard) and not with `CREATE EXTENSION`(this is why it's not listed)

It is currently enabled for the `authenticator` role(the connection role PostgREST uses), so only requests through PostgREST will see the error.

**Comment by t1u1** [comment]
> You enable `safeupdate` with `LOAD 'safeupdate'`(you can do it in the dashboard) and not with `CREATE EXTENSION`(this is why it's not listed)

Ah, ok. I assume it will be loaded by default for all projects. In that case, feel free to close this issue.

Aside, if possible, it might good to show the list of "loaded" extensions on that page.

**Comment by kiwicopple** [comment]
Thanks @t1u1 , I'll add that as a frontend task 👍
